container/features/src/main/resources/features.xml

```diff
@@ -1,1333 +1,1155 @@
+        <feature>twitter4j</feature>
+        <feature>opennms-jaas-login-module</feature>
+        <feature>dashlet-alarms</feature>
+        <feature>dashlet-bsm</feature>
+        <feature>dashlet-charts</feature>
+        <feature>dashlet-image</feature>
+        <feature>dashlet-ksc</feature>
+        <feature>dashlet-map</feature>
+        <feature>dashlet-rrd</feature>
+        <feature>dashlet-rtc</feature>
+        <feature>dashlet-summary</feature>
+        <feature>dashlet-surveillance</feature>
+        <feature>dashlet-topology</feature>
+        <feature>dashlet-url</feature>
+        <feature>dashlet-grafana</feature>
+        <feature>nrtg</feature>
+        <feature>opennms-amqp-event-forwarder</feature>
+        <feature>opennms-amqp-event-receiver</feature>
+        <feature>opennms-amqp-alarm-northbounder</feature>
+        <feature>opennms-topology-runtime-base</feature>
+        <feature>opennms-topology-runtime-browsers</feature>
+        <feature>opennms-topology-runtime-linkd</feature>
+        <feature>opennms-topology-runtime-vmware</feature>
+        <feature>opennms-topology-runtime-application</feature>
+        <feature>opennms-topology-runtime-bsm</feature>
+        <feature>opennms-topology-runtime-graphml</feature>
+        <feature>opennms-topology-runtime-pathoutage</feature>
+        <feature>opennms-topology-runtime-asset</feature>
+        <feature>org.opennms.features.topology.shell</feature>
+        <feature>opennms-enlinkd-shell</feature>
+        <feature>vaadin</feature>
+        <feature>vaadin-dashboard</feature>
+        <feature>vaadin-jmxconfiggenerator</feature>
+        <feature>vaadin-node-maps</feature>
+        <feature>vaadin-snmp-events-and-metrics</feature>
+        <feature>vaadin-surveillance-views</feature>
+        <feature>jira-troubleticketer</feature>
+        <feature>tsrm-troubleticketer</feature>
+        <feature>datachoices</feature>
+        <feature>eif-adapter</feature>
+        <feature>opennms-telemetry-collection</feature>
+        <feature>opennms-telemetry-jti</feature>
+        <feature>opennms-telemetry-nxos</feature>
+        <feature>org.opennms.features.enlinkd.service.api</feature>
+        <feature>org.opennms.features.bsm.service.api</feature>
+        <feature>org.opennms.features.bsm.shell-commands</feature>
+        <feature>vaadin-adminpage</feature>
+        <feature>opennms-jest</feature>
+        <feature>vaadin-opennms-pluginmanager</feature>
+        <feature>alarm-change-notifier</feature>
+        <feature>opennms-es-rest</feature>
+        <feature>internal-plugins-descriptor</feature>
+        <feature>ifttt-integration</feature>
+        <feature>opennms-flows</feature>
+        <feature>opennms-api-layer</feature>
+        <feature>karaf-extender</feature>
+        <feature>opennms-alarm-history-elastic</feature>
+    </feature>
+    <feature name="opennms-integration-api" version="${opennmsApiVersion}" description="OpenNMS Integration API">
```

Comment:
Can we remove this feature, and the sample project one, and reference them from the `<repository>mvn:org.opennms.integration.api/karaf-features/${opennmsApiVersion}/xml</repository>` instead?